common/src/main/java/org/opensearch/sql/common/utils/LogUtils.java

```diff
@@ -42,6 +49,14 @@ public static String getRequestId() {
     return requestId;
   }
 
+  public static void recordProcessingStarted() {
```

Comment:
`LogUtils` was chosen because it currently houses the requestId, which shares a similar responsibility. 
I was going to suggest that we refactor all of this code (and remove the LogUtils class) when we execute on https://bitquill.atlassian.net/browse/AOS-814

We want to choose the earliest available moment to create the start time, but I would have assumed `SQLService` is created when at application startup, not for each request.  But then,  I haven't looked at the class yet.